src/pages/academy/components/search_banner/_topic-items-accordion.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import type { TopicType } from '../../common/_constants'
+import { StyledLink, StyledChevron, HoverChevron } from './_search-styled'
+import { Flex } from 'components/containers'
+import { Header } from 'components/elements'
+import Chevron from 'images/svg/custom/chevron-thick.svg'
+
+type TopicItemsAccordionProps = {
+    items?: TopicType
+    setModal?: React.Dispatch<React.SetStateAction<boolean>>
+    handleHref: (category: string) => void
+}
+
+const DetailsWrapper = styled(Flex)<{ is_expanded: boolean }>`
+    height: auto;
+    overflow: hidden;
+    transition: max-height 0.5s ease;
+    max-height: ${({ is_expanded }) => (is_expanded ? '180px' : '0')};
+
+    :nth-last-child(-n + 2) {
+        max-height: ${({ is_expanded }) => (is_expanded ? '110px' : '0')};
+    }
+`
+
+const TopicItemsAccordion = ({ items, setModal, handleHref }: TopicItemsAccordionProps) => {
+    const [is_expanded, setExpanded] = useState(false)
+
+    const toggleExpand = () => {
+        setExpanded(!is_expanded)
+    }
+
+    const handleModal = () => {
```

Comment:
```suggestion
    const closeModal = () => { ... }
```
the name seems better with `closeModal`, what do you think? 
you can ignore this :pray: